css/style.css

```diff
@@ -16,4 +15,274 @@ body {
+  padding-bottom: 80px;
+}
+.centered-2 {
+  position: absolute;
+  font-size: 2vw;
+  top: 45%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: whitesmoke;
+}
+.centered-btn {
+  position: absolute;
+  top: 70%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: orangered;
+  color: whitesmoke;
+  border: none;
+  padding: 15px;
+  border-radius: 5px;
+}
+.centered-btn:hover {
+  background-color: orange;
+  transition: 0.5s;
+}
+
+.title-section-2 {
+  text-align: center;
+  font-size: 4vw;
+  color: gray;
+  font-weight: 200;
+}
+article section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+article img {
+  width: 125px;
+}
+article {
+  display: flex;
+  justify-content: space-around;
+}
+
+article div p {
+  font-size: 22px;
+  color: gray;
+  font-weight: 500;
+}
+.bottom-content {
+  display: flex;
+  width: 100%;
+}
+.bottom-content img {
+  height: 100%;
+}
+.last-content {
+  display: flex;
+  background-color: pink;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.last-content h3 {
+  text-align: center;
+  font-size: 1.8rem;
+  font-style: italic;
+  font-family: "Harmattan", sans-serif;
+  font-family: "Michroma", sans-serif;
+  letter-spacing: 3px;
+  width: 80%;
+}
+.get-karma-btn {
+  background-color: orangered;
+  color: white;
+  border: none;
+  padding: 10px;
+  border-radius: 5px;
+}
+.get-karma-btn:hover {
+  background-color: orange;
+  transition: 0.5s;
+}
+
+hr {
+  width: 88%;
+  border-top: 2px solid gray;
+}
+
+/*footer*/
+footer p {
+  text-align: center;
+  color: gray;
+}
+.socialMedia {
+  display: flex;
+  justify-content: center;
+  padding: 0;
+}
+.socialMedia a {
+  display: flex;
+  border: 2px solid gray;
+  border-radius: 50px;
+  margin: 10px;
+  width: 50px;
+  height: 50px;
+  justify-content: center;
+  align-items: center;
+}
 
+.socialMedia a img {
+  width: 25px;
+}
+
+/*responsive*/
+
+@media only screen and (max-width: 800px) {
+  .navbar {
+    display: none;
+  }
+  .hamburger {
+    display: flex;
+    align-self: center;
+    padding-right: 3em;
+  }
+  article {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  article section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  article p {
+    text-align: center;
+  }
+
+  .bottom-content {
+    display: flex;
+    flex-direction: column;
+  }
+  .bottom-content img {
+    width: 100%;
+  }
+  .last-content h3 {
+    font-size: 1rem;
+  }
+  .last-content {
+    padding-bottom: 20px;
+  }
+  .get-karma-btn {
+    font-size: 0.5rem;
+  }
+}
+@media only screen and (max-width: 600px) {
+  .centered-1 {
+    position: absolute;
+    font-size: 7vw;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    padding-bottom: 50px;
+    line-height: 0.9em;
+    margin: 0;
+    text-align: center;
+  }
+  .centered-2 {
```

Comment:
Try to give your classes more descriptive names -- for example it could be `.splash-header` and `.splash-subheader` instead of `.centered-1` and `.centered-2`.